qualtricssurvey/qualtricssurvey.py

```diff
@@ -73,9 +85,22 @@ class QualtricsSurvey(StudioEditableXBlockMixin, XBlock):
         'message',
     )
 
+    # pylint: disable=no-member
+    def get_anon_id(self):
+        """
+        Return an anonymous user id
+        """
+        try:
+            user_id = self.xmodule_runtime.anonymous_student_id
+        except AttributeError:
+            user_id = -1
```

Comment:
TODO: something about this...

I think `xmodule_runtime` just isn't available in the workbench?